prac_05/hex_colours.py

```diff
@@ -0,0 +1,15 @@
+HEX_COLOURS = {"aliceblue": "#f0f8ff", "beige": "#f5f5dc", "black": "#000000", "cadetblue": "#5f9ea0",
+               "coral": "#ff7f50", "chocolate": "#d2691e", "forestgreen": "#228b22", "firebrick": "#b22222",
+               "goldenrod": "#daa520", "light": "#eedd82"}
+
+print(HEX_COLOURS)
+
+colour_choice = input("Enter colour: ").lower()
```

Comment:
Could you use the method .casefold() instead of .lower() and then it wouldn't matter if the colours had capital letters in them.  I was playing around with mine and it seemed to work with .casefold().  Just a suggestion :)

Reply:
Sounds interesting, will definitely have a play with that one, thanks.